CSE115A/rideshare-project
# Typeahead Search

Problem
=======
As a user, 
I want to see auto suggestions as I type 
So that I can get more accurate geocodes

[Link to Ticket](https://trello.com/c/K4JJZvta)

Solution
========
Used [`react-select`](https://react-select.com/home) as intake component for the origin and destination addresses.

Change summary:
---------------
* added init type ahead search 3604dc9
* added env var for endpoint 086a37a
* added init typeahead 3c0e6fc
* changing to flex 847cc6a
* added classname to second input d897489
* deleted unneccesary dependencies and added react select e25f00a
* added react select file 4c0a671
* added env var in gh and workflow ea61dfa
* linted yaml file cd7184a

Steps to Verify:
----------------
1. Checkout this branch locally and install dependencies 
1. Start a local dev server and make sure that the two components are shown
1. Type in one component, and ensure that the dropdown shows 5 places as you type 
1. Type in the other component and verify that the dropdown displays 5 suggestions 
1. When clicking on the option, the selected value is saved.

Screenshots:
-----------------------
![Screen-Recording-2020-11-16-at-1](https://user-images.githubusercontent.com/40477553/99306216-b9165f80-2809-11eb-9f7c-1193d4cd3216.gif)
